releasenotes/notes/enable-IMDSv2-by-default-c88a54308df3875f.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    IMDSv2 is now enabled by default. This change should not have any impact.
```

```suggestion
    Change ``ec2_imdsv2_transition_payload_enabled`` from ``false`` to ``true``. 
```

Is there an URL on the datadog public documentation that we can point customers about IMDSv2? If so I would add it in the release notes